ios/Plug/Info.plist

```diff
@@ -20,6 +20,19 @@
 	<string>$(MARKETING_VERSION)</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>co.psychedelic.plug</string>
```

Comment:
maybe we could put `$(PRODUCT_BUNDLE_IDENTIFIER)` here, what do you think @mattgle ?